Sources/Common/Extensions/UIKitExtensions.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+#if canImport(UIKit)
+import UIKit
+
+public extension UIApplication {
+    // since some extension functions may be able to run in iOS app extensions, only disable for this 1 function
+    @available(iOSApplicationExtension, unavailable)
+    func open(url: URL) {
+        open(url, options: [:]) { _ in }
+    }
+}
+
+public extension UIViewController {
+    // find the bundleId of the framework this View belongs in. This can be used to differentiate Views that belong to host app and Views that belong to SDKs/frameworks.
+    var bundleIdOfView: String? {
+        // TODO: create unit test for this function
+        Bundle(for: type(of: self)).bundleIdentifier
+    }
+}
```

Comment:
I am not sure I understand this use case fully, I believe this will (like you have mentioned) not track any 3rd party SDKs either. 

1- We haven't received any requirements for this feature.
2- People might want to track it. for example, someone has an external SDK to handle chat/help screens etc/ All those views will be filtered out as well due to the fact they have different `bundleIdentifier` and might not be using swift ui at all?